README.adoc

```diff
@@ -1343,21 +1343,27 @@ The value for `users.show.title` can be looked up in the template `app/views/use
 
 [source,ruby]
 ----
+# bad
+= t 'users.show.title'
+
+# good
 = t '.title'
 ----
 
 === Dot-separated Keys [[dot-separated-keys]]
 
-Use the dot-separated keys in the controllers and models instead of specifying the `:scope` option.
-The dot-separated call is easier to read and trace the hierarchy.
+Use dot-separated locale keys instead of specifying the `:scope` option with an array or a single symbol.
+Dot-separated notation is easier to read and trace the hierarchy.
 
 [source,ruby]
 ----
 # bad
 I18n.t :record_invalid, scope: [:activerecord, :errors, :messages]
+I18n.t :title, scope: :invitation
 
 # good
 I18n.t 'activerecord.errors.messages.record_invalid'
+I18n.t :record_invalid, scope: 'activerecord.errors.messages'
```

Comment:
Maybe each example should correspond to a `bad` entry?

Reply:
Fixed, thanks 👍